cluster/ci/monitoring/Makefile

```diff
@@ -96,9 +96,11 @@ prow-monitoring-cleanup:
 	oc delete -f ./prometheus_project.yaml --wait=true --ignore-not-found=true --as=system:admin
 
 grafana_password := $(shell uuidgen)
+grafana_deployment := $(shell oc get deploy -n prow-monitoring-stage grafana --no-headers | wc -l)
 
 .PHONY : grafana-debug-deploy
 grafana-debug-deploy:
+ifeq ($(grafana_deployment),0)
```

Comment:
Why do we need this? Shouldn't `oc apply` always work?

Reply:
I feel to avoid unnecessary work to the cluster. :)
But ... because the password is generated on the fly ... so for confusing less people, keep the password unchanged.

Reply:
Hrm. Let's keep the `oc apply` in all cases, it will make sure there are no weird half-updated cases.

Reply:
Done.